docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
+
+This is targeted at Docker 1.7.
+
+# Usage
+
+Plugins can be loaded using a special `docker plugin load` command, as follows:
+
+```
+$ docker plugin load -e $ARGS clusterhq/flocker-plugin $@
```

Comment:
> Should we just pick one of these? Or have neither?

I think having args at the end will be fine for the purpose of this document.

> Having config as part of the plugin image doesn't make sense...

Well, the idea was to reduce the scope of this proposal, there is all sorts of things one can dream up about config, but it's the kind of thing that is always easy to over-engineer. For example, this can easily get everyone into bikeshedding a distributed config store or syntax format, and that's best to avoid if progress is to be made on introducing the primitives, into which this is already getting deep enough.